File: src/main/java/pl/uwr/server/BootstrapData.java
package pl.uwr.server;

import org.springframework.boot.CommandLineRunner;
import org.springframework.stereotype.Component;
import pl.uwr.server.repository.StudentRepository;
import pl.uwr.server.repository.TokenRepository;

@Component
public class BootstrapData implements CommandLineRunner {

    StudentRepository studentRepository;
    TokenRepository tokenRepository;

    public BootstrapData(StudentRepository studentRepository, TokenRepository tokenRepository) {
        this.studentRepository = studentRepository;
        this.tokenRepository = tokenRepository;
    }

    @Override
    public void run(String... args) throws Exception {
        System.out.println("App has started!");
        /*
        Token token = new Token(299008L, Calendar.getInstance(), "esandmoas231mokvsaZASmsadmk232");

        Student siemtomek = new Student(299008L, token);
        token.setStudent(siemtomek);
        studentRepository.save(siemtomek);
        tokenRepository.save(siemtomek.getToken());
        */
        //System.out.println(token.getStudent().getId());
    }
}

File: src/main/java/pl/uwr/server/service/BootstrapData.kt
package pl.uwr.server

import org.springframework.boot.CommandLineRunner
import org.springframework.stereotype.Component
import pl.uwr.server.model.Reservation
import pl.uwr.server.model.Student
import pl.uwr.server.model.Token
import pl.uwr.server.repository.ReservationRepository
import pl.uwr.server.repository.StudentRepository
import pl.uwr.server.repository.TokenRepository
import java.time.DayOfWeek
import java.util.*
import kotlin.jvm.Throws

@Component
/**
 * klasa posiadająca metodę wykonywaną po wystartowaniu serwera
 */
class BootstrapData(private val studentRepository: StudentRepository,private val tokenRepository: TokenRepository) : CommandLineRunner {

    /**
     * metoda uruchamiana po starcie serwera. Może służyć do inicjalizacji zmiennych początkowych
     */
    override fun run(vararg args: String) {

        println("App has started!")
    }

}